pbs/pbsresponse.go

```diff
@@ -24,7 +26,7 @@ func (bids PBSBidSlice) Len() int {
 }
 
 func (bids PBSBidSlice) Less(i, j int) bool {
-	return bids[i].Price < bids[j].Price
+	return bids[i].Price + (float64(bids[i].ResponseTime) / 1000000000.0) < bids[j].Price + (float64(bids[j].ResponseTime) / 1000000000.0)
```

Comment:
only problem with this is that it penalizes the faster response. can we flip to be minus?